python/tests/unittests/test_SpinOperator.py

```diff
@@ -15,7 +15,7 @@
 import cudaq
 from cudaq import spin
 import numpy as np
-
+import scipy 
```

Comment:
Do we need to remove this `import scipy` too if `scipy` is not available?

Reply:
😂 yea I missed the most important part...